packages/vm/src/cache.rs

```diff
@@ -523,8 +536,21 @@ mod tests {
 
         match cache.load_wasm(&checksum).unwrap_err() {
             VmError::CacheErr { msg, .. } => {
```

Comment:
ditto: https://github.com/CosmWasm/wasmvm/pull/338#discussion_r925084121

Reply:
This function gets called at 3 places from `wasmd`:

1. During `ExportGenesis`: https://github.com/CosmWasm/wasmd/blob/82130463ad6d72548aabd7a2e7daf0d2a9b645dc/x/wasm/keeper/genesis.go#L91
2. During `Snapshot`: https://github.com/CosmWasm/wasmd/blob/82130463ad6d72548aabd7a2e7daf0d2a9b645dc/x/wasm/keeper/snapshotter.go#L68
3. During `Query`: https://github.com/CosmWasm/wasmd/blob/82130463ad6d72548aabd7a2e7daf0d2a9b645dc/x/wasm/keeper/querier.go#L284